webpack/assets/javascripts/react_app/redux/actions/templates/index.js

```diff
@@ -0,0 +1,39 @@
+import API from '../../../API';
+import { saveAs } from 'file-saver';
+
+import {
+  TEMPLATE_GENERATE,
+  TEMPLATE_GENERATE_REQUEST,
+  TEMPLATE_GENERATE_POLLING,
+  TEMPLATE_GENERATE_FAILURE,
+} from '../../consts';
+
+const defaultPollingInterval = 2500;
+const pollingInterval =
+  process.env.NOTIFICATIONS_POLLING || defaultPollingInterval;
```

Comment:
this seems like a copy/paste, I wonder if we can generalize it somehow

Reply:
I don't want to introduce another env variable just for polling, so I have reused it. It can be named differently, but we would have to rename it, and it is bit intrusive change :thinking: